update.ts

```diff
@@ -330,3 +330,9 @@ async function findSrcDirectory(
   }
   return resolvedDirectory;
 }
+
+// deno-lint-ignore no-explicit-any
+export function stringifyFormattedJson(jsonStr: any): string {
+  // Ensure formatting rules are met
+  return JSON.stringify(jsonStr, null, 2) + "\n";
+}
```

Comment:
The argument name is a bit misleading because we're not passing in a string.

```suggestion
export function stringifyFormattedJson(data: unknown): string {
  // Append newline to ensure formatting rules are met
  return JSON.stringify(data, null, 2) + "\n";
}
```